.changeset/short-donkeys-double.md

```diff
@@ -0,0 +1,16 @@
+---
+"@comet/cms-admin": minor
+"@comet/admin": minor
+---
+
+Add group section component for usage in menu.
+
+### Example usage
```

Comment:
```suggestion
**Example:**

```

I believe changesets don't support headings. They're rendered in a list. See [releases](https://github.com/vivid-planet/comet/releases) for an example changelog